Seasons Defense/Assets/Scripts/TowerDecider.cs

```diff
@@ -0,0 +1,32 @@
+using UnityEngine;
+
+public class TowerDecider : MonoBehaviour
+{
+    public GameObject leftTurret;
```

Comment:
```suggestion
    public Turret leftTurret;
```
Can we change these references to be of the Turret components themselves so we don't have to use GetComponent in this script, please?

Also, in light of that, can we change the name of TowerController to either Tower or Turret. If we aren't going to use separate Turret and Tower logic, we can just call it `Tower`

Referring to the tower as turret or vice versa makes the code confusing.

Reply:
Yup I have made everything a tower. I can see how it can lead to confusion.